### ISSUE TITLE
Missing train.py

### ISSUE THREAD

**Issue by kkcocoon** [JingZhang-BH/BMFSC#1]
The pytorch code is only for testing, without training process?

**Comment by JingZhang-BH** [comment]
Run classify.py to train.



---- Replied Message ----
| From | Ke-Kun ***@***.***> |
| Date | 01/08/2023 16:03 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [JingZhang-BH/BMFSC] Missing train.py (Issue #1) |

The pytorch code is only for testing, without training process?

**Comment by kkcocoon** [comment]
Thanks. 
I have another question. When comparing test results in target sets (IP,SV, PU) with 5 labeled samples, other methods, such as DCFSL, only use 16*5=80 samples in total on IP for training the fine-tuned network. But for the BMFSC method, each 16-way subtask has 19 queries that are labeled in addition to the 5-shot? Moreover, these 5+19 labeled samples used in each task are still different, is this unfair?

**Comment by JingZhang-BH** [comment]
During the training stage，5+19  samples are selected from meta-training dataset. When traing dataset is the same, the comparasion is fair. That is to say, although the way to use the training dataset is different, the comparasion is fair.



---- Replied Message ----
| From | Ke-Kun ***@***.***> |
| Date | 01/08/2023 17:21 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [JingZhang-BH/BMFSC] Missing train.py (Issue #1) |

Thanks.
I have another question. When comparing test results in target sets (IP,SV, PU) with 5 labeled samples, other methods, such as DCFSL, only use 16*5=80 samples in total on IP for training the fine-tuned network. But for the BMFSC method, each 16-way subtask has 19 queries that are labeled in addition to the 5-shot? Moreover, these 5+19 labeled samples used in each task are still different, is this unfair?

**Comment by kkcocoon** [comment]
Thanks. I see. The trained feature extraction network and the inference network are fixed when testing. In other words, the networks doesn't need to be fine-tuned when testing for IP, SV and PU datasets.  Each task uses 5 labled samples per class as the support set to classify other 19 samples, then calculate the classification accuracy for the 19 samples.
